lib/elf_py.c

```diff
@@ -1140,7 +1140,8 @@ static PyObject *elffile_load(PyTypeObject *type, PyObject *args,
 	fd = open(filename, O_RDONLY | O_NOCTTY);
 	if (fd < 0 || fstat(fd, &st)) {
 		PyErr_SetFromErrnoWithFilename(PyExc_OSError, filename);
-		close(fd);
+		if (fd > 0)
```

Comment:
Ultra pedantic nitpick: `0` is a valid file descriptor, this needs to be `>= 0` :laughing:

(Except that should never happen since 0 is `stdin`…)

Reply:
exaclty ;)